src/libcfitsio.jl

```diff
@@ -208,7 +208,7 @@ function fits_create_file(filename::AbstractString)
     ptr = Array(Ptr{Void}, 1)
     status = Cint[0]
     ccall((:ffinit,libcfitsio), Cint, (Ptr{Ptr{Void}},Ptr{UInt8},Ptr{Cint}),
-          ptr, bytestring(filename), status)
+          ptr, string(filename), status)
```

Comment:
Looking at the old NEWS.md entries, this was necessary on v0.2 but not on v0.3 and later. So I think we can just get rid of these conversions.